static/app/utils/replays/getFrameDetails.tsx

```diff
@@ -74,7 +76,7 @@ const MAPPER_FOR_FRAME: Record<string, (frame) => Details> = {
     color: 'red300',
     description: frame.message,
     tabKey: TabKey.ERRORS,
-    title: defaultTitle(frame),
+    title: <CrumbErrorTitle frame={frame} />,
```

Comment:
@c298lee check this out: by putting this title business up here (and the ui.click description change also in this file) we can avoid having to check the type again inside breadcrumbItem.tsx. 

If breadcrumbItem.tsx were a person it might say:  "I'm going to call getFrameDetails and just print whatever title i get"

Reply:
yea that's a great change :D so much cleaner now!